src/ServiceControl/MessageFailures/FailedMessagesFeature.cs

```diff
@@ -51,15 +53,25 @@ public override void Enrich(IReadOnlyDictionary<string, string> headers, IDictio
                     return;
                 }
 
-                if (isOldRetry)
-                {
-                    bus.Publish<MessageFailureResolvedByRetry>(m => m.FailedMessageId = headers.UniqueId());
-                }
+                var altHeaders = AuditHeadersToFailedMessageHeaders(headers);
```

Comment:
Agreed. I'll update it to just hard-code these Ids

- MessageId, ProcessingEndpoint
- MessageId, ReplyToAddress